.github/workflows/twister-build.yml

```diff
@@ -0,0 +1,59 @@
+name: Twister build workflow
+
+on:
+  workflow_call:
+    inputs:
+      board:
+        required: true
+        type: string
+      manifest:
+        required: true
+        type: string
+      binary_blob:
+        required: false
+        type: string
+
+jobs:
+  twister_build:
+    container:
+      image: zephyrprojectrtos/ci:v0.26.4
+    env:
+      ZEPHYR_SDK_INSTALL_DIR: /opt/toolchains/zephyr-sdk-0.16.1
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository and submodules
+      uses: actions/checkout@v3
```

Comment:
v4?

Reply:
#205 Updates this. I branched that change off of this one so that I could update all the action versions in one place.